gae/room/staff.py

```diff
@@ -962,3 +982,43 @@ def get(self, model_type, model_id):
       self.response.write("model {} does not have a default handler defined in {}".format(model_type, __file__))
       return
     return self.redirect_to(handler.__name__, id=model_id)
+
+
+############################################################################################
+# Following is based on https://cloud.google.com/appengine/docs/standard/python/blobstore/ #
+############################################################################################
+
+class UploadStatementOfWorkAttachment(blobstore_handlers.BlobstoreUploadHandler):
+    def post(self):
+      site_id = self.request.get('site_id')
+      if site_id is None:
+        logging.error("{} did not receive a site_id, nothing to link to".format(self.__class__.__name__))
+        return self.redirect('/room/staff')
```

Comment:
This should 404, so it's clear to user that it's an error condition.

Reply:
Agreed